README.ru.md

```diff
@@ -257,9 +260,9 @@ get '/posts' do
 end
 ```
 
-Кстати, если вы не отключите защиту от обратного пути в директориях (path
-traversal, см. ниже), путь запроса может быть изменен до начала поиска
-подходящего маршрута.
+Имеейте ввиду, что в том случае, если вы не отключите защиту от обратного пути в 
+директориях (''path traversal'', см. ниже), путь запроса может быть изменён до начала 
```

Comment:
why we need `''` here? Oo

Reply:
It's just as it was in previous version of doc, that was _as-is_ translation of original en Readme